crates/bevy_reflect/src/serde/mod.rs

```diff
@@ -16,3 +18,93 @@ pub(crate) mod type_fields {
     pub const ARRAY: &str = "array";
     pub const VALUE: &str = "value";
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{self as bevy_reflect, DynamicTupleStruct};
+    use crate::{
+        serde::{ReflectDeserializer, ReflectSerializer},
+        type_registry::{GetTypeRegistration, TypeRegistry},
+        DynamicStruct, Reflect,
+    };
+    use serde::de::DeserializeSeed;
+
+    #[test]
+    fn test_serialization_struct() {
+        #[derive(Debug, Reflect, PartialEq)]
+        #[reflect(PartialEq)]
+        struct TestStruct {
+            a: i32,
+            #[reflect(ignore)]
+            b: i32,
+            #[reflect(skip_serializing)]
+            c: i32,
+            d: i32,
+        }
+
+        let mut registry = TypeRegistry::default();
+        let registration = TestStruct::get_type_registration();
+        registry.add_registration(registration);
```

Comment:
Super Minor Nit: This could also be written as
```suggestion
        registry.register::<TestStruct>();
```


Same for test below.